force-app/main/default/classes/FeatureParameters.cls

```diff
@@ -60,4 +80,65 @@ public inherited sharing class FeatureParameters {
             return finder.findCount();
         }
     }
+
+    @TestVisible
+    private inherited sharing class ActiveProgramsWithEngagementsLast30 implements FeatureManagement.FeatureParameter {
+        @TestVisible
+        private QueryBuilder queryBuilder {
+            get {
+                if (queryBuilder == null) {
+                    queryBuilder = new QueryBuilder()
+                        .withSObjectType(Program__c.SObjectType)
+                        .withConditions(buildConditions());
+                }
+
+                return queryBuilder;
+            }
+            set;
+        }
+        @TestVisible
+        private Finder finder {
+            get {
+                if (finder == null) {
+                    finder = new Finder(queryBuilder);
+                }
+
+                return finder;
+            }
+            set;
+        }
+
+        public void send() {
+            final Object value = getValue();
+
+            if (value instanceof Integer) {
+                FeatureManagement.getInstance()
+                    .setPackageIntegerValue(getName(), (Integer) value);
+            }
+        }
+
+        private String getName() {
+            return DeveloperName.ACTIVE_PROGRAMS_WITH_ENGAGEMENTS_LAST30.name()
```

Comment:
@jjbennett , curious as to why we are just returning the Active_programs_with_engagements_last30 here. Should we be returning either `Active_programs` or `Active_programs_with_engagements_last30`

Reply:
The name of the featureParameter is ActiveProgramsWithEngagementsLast30. Which is the name of the enum minus the underscores. Does that answer your question, I'm not sure I 100% understood the question.

Reply:
@jjbennett , I was curious as to how the other featureParameter will be used. Is there another getName() method that returns that featureParameter?